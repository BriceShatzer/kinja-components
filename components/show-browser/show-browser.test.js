import * as React from 'react';
import { shallow } from 'enzyme';

import ShowBrowser from './show-browser';

describe('ShowBrowser', () => {
	it('should render', () => {
		const result = shallow(<ShowBrowser
			blogGroup='avclub'
			showMoreButton={false}
			getInitialState={() => Promise.resolve({})}
			loadVideosForTab={() => Promise.resolve([])}
		/>);
		expect(result).toMatchSnapshot();
	});
});
