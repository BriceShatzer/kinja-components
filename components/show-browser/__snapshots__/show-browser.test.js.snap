// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowBrowser should render 1`] = `
<ShowBrowser
  blogGroup="avclub"
  ga={[Function]}
  gaNamespaced={[Function]}
  getInitialState={[Function]}
  loadVideosForTab={[Function]}
  showMoreButton={false}
/>
`;
